src/NewTrackView.elm

```diff
@@ -96,14 +109,14 @@ view model =
                     , onClick AddNewTrack
                     ]
                     [ text "Add New Track" ]
-                , span [ class "prog-form--warning" ] [ text (newTrackWarning model) ]
+                , validationWarningDiv
                 , button
                     [ class "button button--secondary"
```

Comment:
outside the diff - I'm sorry.
Please can we have these take the --wider modifier as well as the save button